utils/build/docker/java_otel/spring-boot-native/src/main/java/com/datadoghq/springbootnative/App.java

```diff
@@ -0,0 +1,53 @@
+package com.datadoghq.springbootnative;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.exporter.logging.otlp.OtlpJsonLoggingSpanExporter;
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackages = {"com.datadoghq.springbootnative"})
+public class App {
+    public static void main(String[] args) {
+        Resource resource = Resource.create(Attributes.of(
+                ResourceAttributes.SERVICE_NAME, "otel-system-tests-spring-boot",
+                ResourceAttributes.DEPLOYMENT_ENVIRONMENT, "system-tests"));
+
+        OtlpHttpSpanExporter intakeExporter = OtlpHttpSpanExporter.builder()
+                .setEndpoint("http://runner:8126/api/v0.2/traces")  // send to the proxy first
+                .addHeader("dd-protocol", "otlp")
+                .addHeader("dd-api-key", System.getenv("DD_API_KEY"))
+                .build();
+
+        SpanExporter loggingSpanExporter = OtlpJsonLoggingSpanExporter.create();
+
+        SpanExporter exporter = SpanExporter.composite(intakeExporter, loggingSpanExporter);
+
+        SpanProcessor processor = BatchSpanProcessor.builder(exporter)
+                .setMaxExportBatchSize(1)
+                .build();
+
+        SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+                .addSpanProcessor(processor)
+                .setSampler(Sampler.alwaysOn())
+                .setResource(resource)
+                .build();
+
+        OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+                .setTracerProvider(sdkTracerProvider)
+                .buildAndRegisterGlobal();
```

Comment:
Why can't we use java auto-instrumentation ?

Reply:
1. System tests need to associate the request id (via user agent header) with span attributes
2. We want to create a fake span and validate span links

Reply:
We can do manual instrumentation along with java-auto instrumentation.  see https://github.com/DataDog/opentelemetry-examples/blob/7725399236180ba5c6fe3f9b9000748341defe38/apps/kafka-redis-messages/otel/calendar-producer-java-otel/src/main/java/com/otel/producer/CalendarProducer.java#L32

Using auto-instrumentation removes lot of boiler plate code and also creates a standard spans

Reply:
Agreed with the benefits of auto-instrumentation, although auto-instrumentation only allows one exporter per signal, which unfortunately doesn't meet with requirements in OTel system tests. In later PRs we'll need to set up 3 span exporters to exercise 3 ingestion paths, that's something auto-instrumentation cannot handle.

Even with manual instrumentation along with auto instrumentation, that won't work because `GlobalOpenTelemetry` (which sets up span exporter and processing) can only be initialized once, i.e. we cannot add more exporters to auto instrumentation.

Reply:
got u . Makes sense.